.circleci/config.yml

```diff
@@ -0,0 +1,188 @@
+defaults:
+  image: &image
+    docker:
+      - image: circleci/node:11.6.0-browsers
+  yarn_cache: &yarn_cache
+    key: yarn_cache-v{{ .Environment.CI_CACHE_VERSION }}-{{ .Branch }}-{{ checksum "yarn.lock" }}
+  build_final_cache_partial: &build_final_cache_partial
+    key: build_cache-v{{ .Environment.CI_CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}-{{ checksum "../workflow-start" }}-all-
+  archive_build: &archive_build
+    name: Archive build
+    command: tar cvf ../build.tar packages/web-components/@sfx/*/{dist,esnext}
```

Comment:
This is fine for now, but this will have to change when we add the wrappers (or anything that's not under `web-components`). If the structure is always going to be `@sfx` under a single subdirectory under `packages`, we should just change this to `packages/*/@sfx/*/{dist,esnext}` now.